### DISCUSSION TITLE
Where to look for help when you don't find the package documentation not useful?

### DISCUSSION THREAD

**Discussion by webdev-dotnet** [community/community#72883] [Programming Help]
### Body

Where to look for help when you don't find the package documentation not useful?

i have been working on some projects lately and use lots of packages, some of which have good documentation but some of them lack the documentation.

where to look for answers of know what to do when the documentation is not good enough but you have to use the package!!!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mamoon-rasheed** [comment]
if the package is new than you can contact the developers and ask for help for certain functionality.
if the package is old and don't have a proper documentation i really want you to avoid using it or else if you have time and energy than look into the code and server yourself.

**Comment by Slax38** [comment]
👍

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
